style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  background-color: darkgrey;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+button {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, 0);
+  border: none;
+  padding:  15px 30px;
+  box-shadow: 0 6px 10px rgba(0, 0, 0, 0.15);
+  cursor: pointer;
+}
+
+.modal {
+  display: none;
+  position: absolute;
+  background: rgba(0, 0, 0, 0.2);
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  justify-content: center;
+  perspective: 500px;
```

Comment:
What do you use `perspecitve` for?
What happens if your remove it?

Reply:
forgot to delete